examples/globe.html

```diff
@@ -44,9 +43,18 @@
         </script>
         <script src="globe.js"></script>
         <script type="text/javascript">
+            /* global itowns, document, GuiTools, globeView, promises */
             const menuGlobe = new GuiTools('menuDiv');
-            new debug.Debug(globeView, viewerDiv);
```

Comment:
Actually yes, I'm not sure they will work on gh-pages because dist is not compiled with debug flag. Good to me.